victims/templates/home.html

```diff
@@ -28,7 +28,8 @@ <h2>
     <h3 class="grey-text">
       {{ case.city }}, {{ case.state }} &mdash; {{ case.when.strftime('%d/%m/%Y') }}
     </h3>
-    <p>{{ case.main_story.summary|e }}</p>
+    <p>{{ case.main_story.summary|e|truncate(max_chars, False, '...') }}</p>
```

Comment:
Can we use the ellipsis character (`…`) instead of three periods (`...`)?

Reply:
done